notebooks/en/fine_tuning_llm_grpo_trl.ipynb

```diff
@@ -0,0 +1,1304 @@
+{
```

Comment:
<p>Let's see how and example looks like &gt;&gt; Let's take a look at an example</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>